web/src/features/charts/elements/AreaGraphLayers.tsx

```diff
@@ -103,7 +105,7 @@ function AreaGraphLayers({
               className={layers.length > 1 ? 'sm:hover:opacity-75' : ''}
               style={{ cursor: 'pointer' }}
               stroke={layer.stroke}
-              fill={isGradient ? `url(#${gradientId})` : layer.fill}
+              fill={isGradient ? `url(#${gradientId})` : layer.fill(layer.key)}
```

Comment:
Which in turn broke this as the URL did not support gradientId with a space in it.

URL encoding also works to fix this issue but it creates gradients for the static colors and inflating the dom. Something we want to avoid.